notebooks/IDC_segmentation_primer.ipynb

```diff
@@ -1,1927 +1 @@
-{
```

Comment:
<p>Perhaps we should also mention loading a manifest file into VolView in this section - as a third option under "I want to have a better visualization" .   Perhaps talk about there being three options.   Does VolView+manifest require a paid-for GCP project?  I thought it could be done using a free account.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ImagingDataCommons/IDC-Tutorials/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='129'/>

Reply:
<p>Added an example</p>

Reply:
<blockquote>Perhaps we should also mention loading a manifest file into VolView in this section - as a third option under "I want to have a better visualization"</blockquote><p> </p><p>We can add that after v14. </p>

Reply:
<p>But note, that in order to visualize analysis results in VolView, the users will have 2 choices:</p><ol><li>store that result in a GCP/AWS storage bucket -&gt; this requires billing set up</li><li>download that result from the notebook to the local file system and drag-and-drop into the VolView tab.</li></ol>

Reply:
<p>I've updated the text to be a bit more explicit that they will need to download the file so that they can drag and drop it. I did not add any information about the GCP/AWS bucket option.</p>